embedly/gunicorn.conf

```diff
@@ -1,5 +1,6 @@
 import multiprocessing
 
+access_logfile = "-"
```

Comment:
I would prefer this to log to /dev/null if it should be dropped. It's easier for me to make sure nothing is kept here, than elsewhere.